.eslintrc.json

```diff
@@ -1,32 +1,62 @@
 {
-    "env": {
-        "browser": true,
-        "es2021": true
-    },
-    "extends": [
-        "google"
-    ],
-    "parser": "@typescript-eslint/parser",
-    "parserOptions": {
-        "ecmaVersion": 12,
-        "sourceType": "module"
-    },
-    "plugins": [
-        "@typescript-eslint",
-        "html"
-    ],
+  "env": {
+    "browser": true,
+    "es2021": true
+  },
+  "extends": [
+    "eslint:recommended",
+    "plugin:cypress/recommended",
+    "google"
+  ],
+  "parser": "@typescript-eslint/parser",
+  "parserOptions": {
+    "ecmaVersion": 11,
+    "sourceType": "module"
+  },
+  "plugins": [
+    "@typescript-eslint",
+    "cypress",
+    "html"
+  ],
   "rules": {
     "max-len": "off",
     "spaced-comment": "off",
     "new-cap": "off",
     "require-jsdoc": "off",
     "no-unused-vars": "off",
-    "@typescript-eslint/no-unused-vars": "error",
     "no-array-constructor": "off",
-    "@typescript-eslint/no-array-constructor": ["error"],
     "valid-jsdoc": "off",
     "comma-dangle": "off",
     "no-invalid-this": "off",
-    "@typescript-eslint/explicit-function-return-type": "error"
-  }
+    // disable TypeScript rule for all files
```

Comment:
hmmm don't these comments break the json?!

Reply:
Hm. You might be right. I saw it like this elsewhere. I will remove them.